_locales/en/messages.json

```diff
@@ -629,6 +685,74 @@
       }
     }
   },
+  "styleInstallOverwrite" : {
+    "message": "'$stylename$' is already installed. Overwrite?\nVersion: $oldVersion$ -> $newVersion$",
+    "description": "Confirmation when re-installing a style",
+    "placeholders": {
+      "stylename": {
+        "content": "$1"
+      },
+      "oldVersion": {
+        "content": "$2"
+      },
+      "newVersion": {
+        "content": "$3"
+      }
+    }
+  },
+  "styleInstallNoName": {
+    "message": "Install this style into stylus?",
```

Comment:
I think Stylus should be capitalized here. We're only using lowercase naming for the repository... or maybe I'm wrong?

Reply:
This should be removed, we don't use it after implementing the install page.